### PR TITLE
Adding config parameters like url, interval and frequency

### PR DESCRIPTION
**Pull request by YashviJain01**
### :nut_and_bolt: Description: What code changed, and why?

Add the license audit scheduler parameter in config.

### :chains: Related Resources

https://chefio.atlassian.net/browse/CHEF-5804

### :+1: Definition of Done

Now the schedule can be changed for testing purpose from the config itself.

### :athletic_shoe: How to Build and Test the Change

The steps are mentioned in the demo video

```
components/license-control-service/

cd /hab/pkgs/<origin>/license-control-service/<pkg-path>/
add change in default.toml
```

### :white_check_mark: Checklist

**All PRs** must tick these:

- [X] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [X] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [X] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [X] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [X] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [X] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [X] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [X] Tests added/updated? (all new code needs new tests)
- [X] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### :camera: Screenshots, if applicable

https://progresssoftware.sharepoint.com/:v:/s/ChefCoreC/EUqlG7DK50RDm4UinQ1oNEEBN_HnMvfESD7tu4blm9xINQ?e=q9jX3N

After hour support:-
image

### CODE REVIEW

File: components/license-control-service/licenseaudit/licenseauditworkflow.go [hunk 1]
```diff
@@ -47,18 +47,32 @@ func InitCerealManager(ctx context.Context, cerealManager *cereal.Manager, worke
...
-		Freq:     rrule.DAILY,
-		Interval: 1,
+		Freq:     getInterval(frequency),
+		Interval: interval,
 		Dtstart:  modifiedTime,
 	})
 
 }
 
+func getInterval(frequency string) rrule.Frequency {
```
Comment (Dmaddu) [inline components/license-control-service/licenseaudit/licenseauditworkflow.go:62]: Are we not supporting hourly?

**Review by Dmaddu — CHANGES_REQUESTED** [review]

File: components/license-control-service/licenseaudit/licenseauditworkflow.go (hunk 1 above)
Comment (YashviJain01) [reply components/license-control-service/licenseaudit/licenseauditworkflow.go:62]: We can add, let me do that

Comment (YashviJain01) [reply components/license-control-service/licenseaudit/licenseauditworkflow.go:62]: Addded.

**Review by Dmaddu — APPROVED** [review]

**Comment by Dmaddu** [comment]
Hi @YashviJain01, Since you are adding config parameters, it would be good if you could also provide the demo of patching those parameters. Please make to include these things in the demo next time onwards. Approving for now.

**Pull request merged**